source/libs/page-detect.ts

```diff
@@ -373,6 +374,14 @@ export const _isBranches = [
 export const isUserProfile = (): boolean => select.exists('.user-profile-nav');
 export const _isUserProfile = domBased;
 
+export const isUserProfileRepoTab = (): boolean => isUserProfile() &&
+	new URLSearchParams(location.search).get('tab') === 'repositories' &&
+	location.pathname !== '/'; // Gotcha for 'https://github.com/?tab=repositories'
```

Comment:
?

This is already covered by isUserProfile

Reply:
I know. My AVA is not working locally (And my computer is doing funny things) I Forgot to remove it

Reply:
What's not working? Make sure you test this on Node 12.15. You can use [n](https://www.npmjs.com/package/n) to switch between versions

Reply:
'TS_NODE_FILES' is not recognized as an internal or external command

Reply:
Are you on Windows 10? It should work on bash

Reply:
> Are you on Windows 10? It should work on bash

Yes. Let me try

Reply:
Or, more easily for now, first set the `TS_NODE_FILES` environment variable and then just run `npx ava` directly, not `npm run test:js`

Reply:
```console
ava

  
  × Timed out while running tests

  × No tests found in test\search-query.ts
  × No tests found in test\page-detect.ts
  × No tests found in test\utils.ts

```

Give on trying to fix it

Reply:
This happens to me sometimes too, it's because ava tries to run all the tests in parallel and times out. Sometimes I just try again and it works.

Try `ava --serial`.